src/DependencyInjection/HttplugExtension.php

```diff
@@ -292,6 +294,24 @@ private function configurePluginByName($name, Definition $definition, array $con
 
                 break;
 
+            case 'throttle':
+                if (!\class_exists(ThrottlePlugin::class)) {
+                    throw new InvalidConfigurationException('You need to require the Throttle Plugin to be able to use it: "composer require php-http/throttle-plugin".');
+                }
+
+                $key = $config['name'] ? '.'.$config['name'] : '';
+                $container
+                    ->register($serviceId.$key, LimiterInterface::class)
+                    ->setFactory([new Reference('limiter.'.$config['name']), 'create'])
```

Comment:
this does not check if the name is set or not. and if the name is empty, it will use `limiter.`, is that the intention?

Reply:
@dbu No, it was my mistake, I assume that if the default value is not specified, then it is required by default